docs/releasing-files.md

```diff
@@ -252,6 +252,24 @@ Only certain file types will be reviewed and released from the secure server. Se
 
 If you would like to release other file types, please email datarelease@opensafely.org, stating why it is important that the file is released in a different format.
 
+### Checklist
+
+Please run through this checklist before making a review request.
+
+1. Do your results adhere to the [OpenSAFELY permitted study results policy](https://www.opensafely.org/policies-for-researchers/#permitted-study-results-policy)
+2. Are all of the outputs of the [allowed file types](#allowed-file-types)?
+3. Are all of the outputs in a [separate release folder](#2-requesting-release-of-outputs-and-error-log-files-from-the-server)?
+4. Have you [redacted any low counts](#redacting-counts-less-than-or-equal-to-5)? 
+5. Have you [rounded any counts](#redacting-counts-less-than-or-equal-to-5) (including [counts underlying rates](#rounding-rates))?
+6. Have you suplpied underlying counts for any tables/figures?
```

Comment:
Underlying counts for tables may be confusing - how does one know if a table needs further underlying counts to be supplied?

Reply:
I've changes this to just say "underlying counts for any results"